contracts/core/TribeRoles.sol

```diff
@@ -48,19 +48,28 @@ library TribeRoles {
     /// @notice manages TribalChief incentives and related functionality.
     bytes32 internal constant TRIBAL_CHIEF_ADMIN =
         keccak256("TRIBAL_CHIEF_ADMIN_ROLE");
+    bytes32 internal constant TOKEMAK_DEPOSIT_ADMIN_ROLE =
+        keccak256("TOKEMAK_DEPOSIT_ADMIN_ROLE");
 
     /// @notice admin of PCVGuardian
     bytes32 internal constant PCV_GUARDIAN_ADMIN =
         keccak256("PCV_GUARDIAN_ADMIN_ROLE");
 
+    /// @notice admin of the peg stability modules
+    bytes32 internal constant PSM_ADMIN_ROLE = keccak256("PSM_ADMIN_ROLE");
+
     /// @notice admin of all Minor Roles
     bytes32 internal constant MINOR_ROLE_ADMIN = keccak256("MINOR_ROLE_ADMIN");
 
     /// @notice admin of the Fuse protocol
     bytes32 internal constant FUSE_ADMIN = keccak256("FUSE_ADMIN");
 
-    /// @notice capable of vetoing DAO votes or optimistic timelocks
-    bytes32 internal constant VETO_ADMIN = keccak256("VETO_ADMIN");
```

Comment:
This was a pre-existing role from before the governance upgrade. We now have `POD_VETO_ADMIN` instead